horizon/opal_relay_api.py

```diff
@@ -122,19 +138,39 @@ async def send_ping(self):
             ),
         ) as response:
             if response.status != status.HTTP_202_ACCEPTED:
-                text = await response.text()
-                raise RuntimeError(
-                    f"Server responded to token request with a bad status {response.status}: {text}"
+                try:
+                    text = await response.text()
+                except:
+                    text = None
+                raise RelayAPIException(
+                    "relay-api",
+                    response.status,
+                    f"Server responded to token request with a bad status {response.status}: {text}",
                 )
         logger.debug("Sent ping.")
 
     async def _run(self):
+        backoff = 0
```

Comment:
why did you add a backoff functionality?
- this will make resending failed updates much much slower. we don't want that.